app/src/main/AndroidManifest.xml

```diff
@@ -39,6 +39,17 @@
       android:name="com.mixpanel.android.MPConfig.DisableViewCrawler"
       android:value="true" />
 
+    <activity
+      android:name=".setup.SetupActivity"
+      android:enabled="false"
```

Comment:
I assume that since the `SetupActivity` is disabled, we will still continue launching `TheActivity`. Is this a safe assumption to make?

Reply:
Yup, the final PR will change `SetupActivity` to be the only launcher.